tasks/deploy/config/mainnet/morpho-blue-wsteth-usda-182day.ts

```diff
@@ -0,0 +1,126 @@
+import { Address, encodeAbiParameters, parseEther, zeroAddress } from "viem";
+import {
+    HyperdriveInstanceConfig,
+    SIX_MONTHS,
+    USDA_ADDRESS_MAINNET,
+    getLinkerDetails,
+    normalizeFee,
+    parseDuration,
+    toBytes32,
+} from "../../lib";
+import { MAINNET_FACTORY_NAME } from "./factory";
+import { MAINNET_MORPHO_BLUE_COORDINATOR_NAME } from "./morpho-blue-coordinator";
+
+export const MAINNET_MORPHO_BLUE_WSTETH_USDA_182DAY_NAME =
+    "ElementDAO 182 Day Morpho Blue WSTETH/USDA Hyperdrive";
```

Comment:
is `WSTETH` consistent with other naming? For gnosis we used `wstETH`....

Reply:
also USDA vs USDa

Reply:
I haven't seen `USDa` anywhere. If you look on their website, they use `USDA`: https://www.angle.money/usda. I will update `wstETH`